R/R/clusters.R

```diff
@@ -331,12 +332,18 @@ errors_scores <- function(df, balance = rep(1, 3), ts_validation = TRUE, ...) {
 
 .min_max_norm <- function(x, min = 0, max = 1) {
   x <- x[is.finite(x)]
-  if (length(x) == 1) {
+  x <- x[!is.na(x)]
+  if (length(x) <= 1) {
     return(x)
-  } # return((max - min) / 2)
+  }
   a <- min(x, na.rm = TRUE)
   b <- max(x, na.rm = TRUE)
-  (max - min) * (x - a) / (b - a) + min
+  den <- ((b - a) + min)
+  if (den != 0) {
+    return((max - min) * (x - a) / den)
```

Comment:
Yes, I just want to avoid dividing by zero. That's the logic behind that conditional @gufengzhou with your example, if let's say b-a=1 and min is 1, you'll get an Inf. With my proposal, I'm avoiding that.

Reply:
Hm i don't think so. The order of operation dictates that multiplication and division come first, then addition. So the +min is not in the denominator. Or does R follows a different order?!

Reply:
Ugh, obviously! Fixed